statik.go

```diff
@@ -519,6 +525,12 @@ func main() {
 	debug := flag.Bool("d", false, "Print debug logs")
 	flag.Parse()
 
+	if *debug {
```

Comment:
Non vengono mai fatte chiamate di debug, che io sappia, mi sembra inutile.

Reply:
Di default `log.Print` usa debug. Se vedi riga 577-585 utilizza `log.Print`.

Reply:
Okay, il comportamento è come una volta allora. Si potrebbero aggiungere logging in giro in modalità debug, ma pace.